redaxo/src/addons/structure/plugins/content/lib/article_content_editor.php

```diff
@@ -77,14 +77,16 @@ protected function outputSlice(rex_sql $artDataSql, $moduleIdToAdd)
                     $moduleInput = $this->replaceVars($artDataSql, $moduleInput);
                     return $sliceContent . $this->editSlice($sliceId, $moduleInput, $sliceCtype, $moduleId, $artDataSql);
                 }
-                // Modulinhalt ausgeben
-                $moduleOutput = $this->replaceVars($artDataSql, $moduleOutput);
-                $panel .= $this->getWrappedModuleOutput($moduleId, $moduleOutput);
-            } else {
-                // ----- hat keine rechte an diesem modul, einfach ausgeben
-                $moduleOutput = $this->replaceVars($artDataSql, $moduleOutput);
-                $panel .= $this->getWrappedModuleOutput($moduleId, $moduleOutput);
             }
+            // Modulinhalt ausgeben
+            $moduleOutput = $this->replaceVars($artDataSql, $moduleOutput);
+            $content = $this->getWrappedModuleOutput($moduleId, $moduleOutput);
+
+            // EP for changing the module preview
+            $panel .= rex_extension::registerPoint(new rex_extension_point('SLICE_BE_PREVIEW', $content, [
+                'moduleOutput' => $moduleOutput,
+                'moduleId' => $moduleId,
```

Comment:
Sorry, ist mir erst jetzt aufgefallen: Bei den Params in den EPs nutzen wir `snake_case`, also `module_id`.

Und Frage zu `moduleOutput`: Hast du konkreten Bedarf für den Param? Denn der enthält ja den noch nicht ausgeführten PHP-Code. Den string-basiert zu manipulieren ist also besonders problematisch.

Reply:
@gharlan ist beides korrigiert, moduleOutput brauchen wir glaub ich nicht.